frame/instrumental-strategy-pablo/src/mock/helpers.rs

```diff
@@ -0,0 +1,35 @@
+use composable_traits::defi::CurrencyPair;
+use frame_support::assert_ok;
+use pallet_pablo::PoolInitConfiguration;
+use primitives::currency::CurrencyId;
+use sp_runtime::Permill;
+
+use super::runtime::{Balance, Pablo, PoolId};
+use crate::mock::runtime::{AccountId, BlockNumber};
+
+pub fn create_usdt_usdc_pool() -> PoolId {
```

Comment:
Why you choose stable assets?

Reply:
See comment [above](https://github.com/ComposableFi/composable/pull/1195#discussion_r904895981).